### DISCUSSION TITLE
Improve clarity of UI showing time span

### DISCUSSION THREAD

**Discussion by gene1wood** [lukevella/rallly#133] [Ideas]
![Selection_365](https://user-images.githubusercontent.com/1134034/164762717-4888149e-de31-490c-bb13-55651deb2b11.png)

I think it would be helpful if something was added to communicate that the two times that are next to each other (in the image `12:00 PM` and `1:00 PM`) represent the start and end time of a span of time.

When I first saw them, especially in a grid of 4 times if you have two dates, I didn't understand that they were start and end times. After playing around with it I was able to figure it out.

Something visual like a line, or the words start and end maybe?

![Selection_365](https://user-images.githubusercontent.com/1134034/164763123-e8e32619-d5ac-4cdf-be22-a6fc8bfa1ecd.png)

**Comment by lukevella** [comment]
Hi @gene1wood, moving this to discussions since it's a more appropriate place for this kind of thing. I agree with you in principle and you can expect to see a refresh of the poll creation screen soon that should address this.

**Comment by heyakyra** [comment]
Selecting times to offer as options gives a nice grid…but it's a nightmare for users who have to scroll horizontally across a bunch of individual items. Why not offer the ability to RSVP on a similar grid to the managament UI? 

Ideally, there would be a dedicated mode to scheduling a weekly (or other regular) timeslot based on a dateless timetable. Examples being https://lettucemeet.com/ and http://whenisgood.net/
